packages/meditrak-server/src/routes/changePassword.js

```diff
@@ -7,7 +7,19 @@ import { editRecord } from './editRecord';
 
 export async function changePassword(req, res, next) {
   const { models, body, userId } = req;
-  const { oneTimeLoginToken, oldPassword, password, passwordConfirm } = body;
+  const {
+    oneTimeLoginToken,
+    oldPassword,
+    password,
+    newPassword,
+    passwordConfirm,
+    newPasswordConfirm,
+  } = body;
+
+  // Support both alternatives so that users using old versions
```

Comment:
```suggestion
  // Support both alternatives so that users using versions of meditrak-app prior to 1.9.109
```
Have found it's always handy to be specific, so that reading over it in future we will know when to deprecate